src/components/NewGalaxyModal.js

```diff
@@ -37,394 +37,397 @@ const maxNodepoolSize = 1000 // per zone according to https://cloud.google.com/k
 // Removing low cpu/memory options based on the Galaxy team's suggestions
 const validMachineTypes = _.filter(({ cpu, memory }) => cpu >= 4 && memory >= 52, machineTypes)
 
-export const NewGalaxyModal = _.flow(
```

Comment:
most changes to this file are indention changes since the top level flow was changed. I will call out the actual changes